src/components/dataset/query/sidebar/panels/InfoView.jsx

```diff
@@ -87,14 +149,22 @@ export class InfoView extends React.PureComponent {
         )}
         {!isHelpVisible && (
           <Grid container spacing={2}>
-            <Grid item xs={12}>
-              <Typography variant="h4">Dataset Overview</Typography>
-            </Grid>
             <Grid item xs={12}>
               <Typography variant="h5">{dataset.name}</Typography>
             </Grid>
+            {linkToBq && (
+              <Grid item xs={12}>
+                <TerraTooltip title="Click to navigate to the Google BigQuery console where you can perform more advanced queries against your dataset tables">
+                  <Button color="primary" variant="contained" disableElevation endIcon={<Launch />}>
```

Comment:
This button doesn't seem to be aligned with the `Tables` header. Might not be important now, since as you've said, this is going away.